doc/mgr/dashboard.rst

```diff
@@ -65,31 +66,32 @@ aspects of your Ceph cluster:
   additional information and performance metrics gathered by the
   :ref:`mgr-prometheus`. See :ref:`dashboard-grafana` for details on how to
   configure this functionality.
-* **Cluster logs**: Display the latest updates to the cluster's event and audit
-  log files.
+* **Cluster logs**: Displays the latest updates to the cluster's event and
+  audit log files. Log entries can be filtered by priority, date or keyword.
 * **Hosts**: Provides a list of all hosts associated to the cluster, which
   services are running and which version of Ceph is installed.
 * **Performance counters**: Displays detailed service-specific statistics for
   each running service.
 * **Monitors**: Lists all MONs, their quorum status, open sessions.
-* **Configuration Editor**: View all available configuration options,
+* **Configuration Editor**: Displays all available configuration options,
   their description, type and default values and edit the current values.
-* **Pools**: List all Ceph pools and their details (e.g. applications, placement
-  groups, replication size, EC profile, CRUSH ruleset, etc.)
+* **Pools**: Lists all Ceph pools and their details (e.g. applications,
+  placement groups, replication size, EC profile, CRUSH ruleset, etc.)
 * **OSDs**: Lists all OSDs, their status and usage statistics as well as
-  detailed information like attributes (OSD map), metadata, performance counters
-  and usage histograms for read/write operations. Mark OSDs as up/down/out,
-  perform scrub operations. Select between different recovery profiles to adjust
-  the level of backfilling activity.
+  detailed information like attributes (OSD map), metadata, performance
+  counters and usage histograms for read/write operations. OSDs can be marked
+  as up/down/out, scrub operations can be performed. Different recovery
+  profiles can be selected to adjust the level of backfilling activity.
 * **iSCSI**: Lists all hosts that run the TCMU runner service, displaying all
   images and their performance characteristics (read/write ops, traffic).
   Create, modify and delete iSCSI targets (via ``ceph-iscsi``). See
   :ref:`dashboard-iscsi-management` for instructions on how to configure this
   feature.
 * **RBD**: Lists all RBD images and their properties (size, objects, features).
-  Create, copy, modify and delete RBD images. Create, delete and rollback
-  snapshots of selected images, protect/unprotect these snapshots against
-  modification. Copy or clone snapshots, flatten cloned images.
+  Create, copy, modify and delete RBD images. Define various I/O or bandwidth
+  limitation settings on a per-pool or per-image level. Create, delete and
```

Comment:
> settings on a per-pool or per-image level

It is possible to do that on a global level, too, but that page isn't as comfortable as the others. You'd have to go to "Cluster -> Configuration", then change "Level" to "Advanced". The config options can then be found by its key (e.g. `rbd_qos_bps_burst`) and its value can be changed. But because it isn't convenient, I'm not sure if we should advertise it. Nonetheless, it's possible using the dashboard.

Reply:
Thanks for pointing this out. I've added "global" to the possible levels at which these settings can be changed, without further elaborating.